tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -793,18 +818,32 @@ func getIngressOrFail(t *testing.T) string {
 
 // TestCheckCache tests that check cache works within the mesh.
 func TestCheckCache(t *testing.T) {
+	const (
```

Comment:
Move to top of file so that 'remote', 'local' can be used by the other functions.

Reply:
OK, I've now entered nit territory. Perhaps using 'remote_cluster', 'local_cluster' would make this more readable. If implemented, change my comment on podID() variable from 'pod_location' to 'cluster_location'.